README.md

```diff
@@ -19,12 +19,24 @@ config = {
     'LINK' => DraftjsExporter::Entities::Link.new(className: 'link')
   },
   block_map: {
-    'header-one' => { element: 'h1' },
+    'header-one' => { element: 'h1', attrs: { id: 'foo-bar' }},
     'unordered-list-item' => {
       element: 'li',
       wrapper: ['ul', { className: 'public-DraftStyleDefault-ul' }]
     },
-    'unstyled' => { element: 'div' }
+    'unstyled' => { element: 'div' },
+    # For atomic blocks, we match data to find the right block blocks.
+    # Unmatched atomic blocks will use the un-styled block options.
+    'atomic' => [
```

Comment:
As a consumer of this gem, I'd also expect to be able to pass a straight object in (as opposed to always having to pass an array).